### PR TITLE
Updated ARC related profiles

### PR DESCRIPTION
**Pull request by floWetzels**
Updated ARC related profiles, since some aspects within them where outdated. Also added a new sub-profile for the CWL part of ARCs (extension of workflow run crate, see https://github.com/nfdi4plants/arc-cwl-ro-crate-profile). For easier maintenance, we decided to remove the explicit part on the RO-Crate website and only refer to the profile's repository.

### CODE REVIEW

File: docs/pages/resources/profiles.md [hunk 1]
```diff
@@ -322,108 +322,28 @@ evolved from this profile to conform with RO-Crate 1.1's
...
 An ARC consists of [ISA](https://isa-specs.readthedocs.io/en/latest/isamodel.html) metadata describing the experimental setup and computational workflows given in CWL.
-The current [profile](https://github.com/nfdi4plants/ARC-specification/blob/main/ARC%20specification.md#appendix-conversion-of-arcs-to-ro-crates) requires the crate to follow the ISA Investigation profile on the top level.
-In the future, the ARC profile will be extended to not only cover the ISA part of an ARC, but also computational workflows, following the existing profiles for this kind of data.
+The current profile is a combination of the [ISA RO-Crate profile](https://github.com/nfdi4plants/isa-ro-crate-profile) and the [ARC CWL RO-Crate profile](https://github.com/nfdi4plants/arc-cwl-ro-crate-profile), see details below.
+It requires the crate to follow the ISA Investigation profile on the top level.
+The investigation object contains studies and assays from the ISA model, as well as computational workflows.
+The core entity of both sub-profiles is the type `LabProcess` defined by [bioschmeas](https://bioschemas.org/) and used to describe processes leading to data files in a parametrized way.
```
Comment (elichad) [inline docs/pages/resources/profiles.md:330]:
```suggestion
The core entity of both sub-profiles is the type `LabProcess` defined by [Bioschemas](https://bioschemas.org/) and is used to describe processes leading to data files in a parametrized way.
```

File: docs/pages/resources/profiles.md [hunk 2]
```diff
@@ -322,108 +322,28 @@ evolved from this profile to conform with RO-Crate 1.1's
...
-  - [`sameAs`](http://schema.org/sameAs): [`URL`](https://schema.org/URL) (required)
-  - [`headline`](http://schema.org/headline): [`Text`](https://schema.org/Text) (required)
-  - [`author`](http://schema.org/author): [`Person`](https://schema.org/Person) (required)
-   
-  - [`url`](http://schema.org/url): [`URL`](https://schema.org/URL) (recommended)
-   
-  - [`creativeWorkStatus`](http://schema.org/creativeWorkStatus): [`DefinedTerm`](https://schema.org/DefinedTerm) (optional)
-  - [`disambiguatingDescription`](http://schema.org/disambiguatingDescription): [`Text`](https://schema.org/Text) (optional) -->
+The profile adds requirements of the crate such that the data folders match the Investigation, Study and Assay objects of the [ISA model](https://isa-specs.readthedocs.io/en/latest/isamodel.html).
+The description of experimental processes that produce the data uses the [bioschmeas](https://bioschemas.org/) type `LabProcess` as its core entity.
```
Comment (elichad) [inline docs/pages/resources/profiles.md:339]:
```suggestion
The description of experimental processes that produce the data uses the [Bioschemas](https://bioschemas.org/) type `LabProcess` as its core entity.
```

File: docs/pages/resources/profiles.md [hunk 3]
```diff
@@ -322,108 +322,28 @@ evolved from this profile to conform with RO-Crate 1.1's
...
+It describes how data and samples are derived from each other in a graph model based on formalized parameters as key-value pairs.
+Profiles for all included types can be found in the [full version](https://github.com/nfdi4plants/isa-ro-crate-profile).
+
+## ARC CWL RO-Crate Profile
+
+A [ARC CWL RO-Crate profile](https://github.com/nfdi4plants/arc-cwl-ro-crate-profile) extends the existing Worflow Run Crate in an ARC-specific context.
```
Comment (elichad) [inline docs/pages/resources/profiles.md:345]:
```suggestion
The [ARC CWL RO-Crate profile](https://github.com/nfdi4plants/arc-cwl-ro-crate-profile) extends the existing [Workflow Run Crate profile](https://w3id.org/ro/wfrun/workflow/) in an ARC-specific context.
```

File: docs/pages/resources/profiles.md [hunk 4]
```diff
@@ -322,108 +322,28 @@ evolved from this profile to conform with RO-Crate 1.1's
...
+It uses the same process graph types (`LabProtocol`,`LabProcess`) as the ISA profile to further annotate parameters of workflows (equivalent to protocols in ISA) and runs (equivalent to processes in ISA).
```
Comment (elichad) [inline docs/pages/resources/profiles.md:346]:
```suggestion
It uses the same process graph types (`LabProtocol`,`LabProcess`) as the [ISA profile](#isa-ro-crate-profile) to further annotate parameters of workflows (equivalent to protocols in ISA) and runs (equivalent to processes in ISA).  
```

**Review by elichad — COMMENTED** [review]
Thanks @floWetzels, this is a nice update! I've made suggestions to fix typos and add a couple of extra links, once those are committed I will approve and merge

File: docs/pages/resources/profiles.md (hunk 1 above)
Comment (floWetzels) [reply docs/pages/resources/profiles.md:330]: To me, the second 'is' seems wrong here. Maybe change it to "The core entity of both sub-profiles is the type `LabProcess` defined by Bioschemas, which is used to describe processes leading to data files in a parametrized way"?

Comment (elichad) [reply docs/pages/resources/profiles.md:330]: sure, updated!

**Review by elichad — APPROVED** [review]

**Pull request merged**